package.json

```diff
@@ -10,8 +10,9 @@
     "start": "concurrently \"NODE_ENV=dev webpack --watch --config config/server.js\" \"php -S localhost:9090\" --kill-others",
     "dev": "NODE_ENV=dev webpack --watch --config config/webpack.dev.js",
     "prod": "webpack --config config/webpack.prod.js && npm run favicon && NODE_ENV=dev webpack --config config/webpack.dev.js",
-    "build": "npm run prod && node tasks/critical-css.js && node tasks/style-version.js && cd livingcss && npm install && gulp livingcss",
+    "build": "npm run prod && node tasks/critical-css.js && node tasks/style-version.js && node tasks/svg-sprite.js && cd livingcss && npm install && gulp livingcss",
```

Comment:
@Nkzq npm run icons plutot que node tasks/svg-sprite.Sjs non ?
sinon j'ai  TypeError: svgSprite is not a function lorsque j'ajoute/retire un icon :/

Reply:
Oui j'ai rencontré quelques soucis, notamment le npm run icons qui crash le serveur. Il faut que je commit mes correctifs.

Reply:
j'ai ajusté les path @Nkzq tu regardes de ton coté si c'est ok ? pour moi c'est mergeable, faudrait rajouter un watch sur `src/img/icons/*.svg` pour `npm run icons`